doc/policy/packages.md

```diff
@@ -36,10 +36,9 @@ The following rules are enforced for all packages:
 * Packages cannot have conflicting transactions, i.e. no two transactions in a package can spend
    the same inputs. Packages cannot have duplicate transactions. (#20833)
 
-* No transaction in a package can conflict with a mempool transaction. Replace By Fee is
-  currently disabled for packages. (#20833)
+* Only limited package replacements are currently considered. (#28984)
 
-   - Package RBF may be enabled in the future.
+   - More general package RBF may be enabled in the future.
```

Comment:
Could add more detailed docs for what the rules are in package RBF?

- all to-be-replaced signal (bip125 or v3)
- package is 1p1c, only in clusters up to size 2
- to-be-replaced all in clusters up to size 2
- no more than 100 total to-be-replaced
- total fees of package > all fees being replaced, at incremental relay feerate
- must improve feerate diagram
- parent feerate must be < package feerate

Reply:
@glozow pushed some more detailed docs, thanks!